docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+Neptune-Keras Integration
+*************************
+
+What will you get with this integration
+=======================================
+
+In this tutorial, you'll learn how to integrate Neptune into your Keras application, and at the end you'll be able to efficiently track and monitor model training, metrics, images and even datasets.
+
+Video [1min screencast]
+
+|Keras| is a high level interactive API that can be used to prototype, build and train neural networks. \
+Keras acts as an interface for the TensorFlow library, and in the latest version of Tensorflow,\
+Keras was officially integrated under the name |tf.keras|. The original Keras is still available,\
+but in this tutorial, you'll be using the Tensorflow version \
+of Keras (TF Keras). Don't worry, the API is totally the same. For more explanation \
+on TF Keras, check out this |link|. At the end of this tutorial you will learn: \
+
+* How to integrate Neptune with Keras to effectively track metrics and model training. 
+* How to easily reproduce experiments and aid collaboration in your project. 
+
+**Note:** This integration works with library==tf.keras v0.24.1 and neptune-contrib v0.24.1 
```

Comment:
Let's use the appropriate Note syntax:

```
.. note::

    This ....
```

Also let's just have library version as tf.keras==0.24.1 and neptune-contrib==0.24.1 
what was the neptune-client version you were using?